### DISCUSSION TITLE
Help to run example

### DISCUSSION THREAD

**Discussion by marcelK0** [spali/go-rscp#48] [General]
Hi,
really excited about your work!

I'm having issues running your example though.

I installed go 1.21 on linux and trying to execute
`go run ./e3dc.go`
but keep getting

```
./e3dc.go:15:16: undefined: conf
./e3dc.go:16:23: undefined: conf
./e3dc.go:17:16: undefined: conf
./e3dc.go:18:16: undefined: conf
./e3dc.go:19:16: undefined: conf
./e3dc.go:31:15: undefined: unmarshalJSONRequests
./e3dc.go:31:44: undefined: conf
./e3dc.go:34:5: undefined: conf
./e3dc.go:46:9: undefined: conf
./e3dc.go:52:29: undefined: NewJSONSimpleMessages
./e3dc.go:52:29: too many errors
```

I did put a `.config` file in the local directory. But also it seems that those JSON methods are not being recognised. Could you please provide a few more details on how to execute the example exactly.

Many thanks!

**Comment by spali** [comment]
You need to run the actual executable package in cmd/e3dc from the root of the project.

**Reply by marcelK0** [reply]
Ah ok, just running `go build` in cmd/e3dc compiled the binary fine. Thx
